Dixit_ServiceLibrary/Interfaces/IDixitService.cs

```diff
@@ -24,7 +24,7 @@ public interface IDixitService
         /// </summary>
         /// <returns>Jatek lista.</returns>
         [OperationContract(IsInitiating = false, IsTerminating = false)]
-        Game[] ListGames();
+        IDixitGame[] ListGames();
```

Comment:
Can we use List<IDixitGame> here? I think collection api is a lot friendlier than an array.